sync/class.jetpack-sync-module-posts.php

```diff
@@ -228,7 +234,12 @@ public function save_published( $new_status, $old_status, $post ) {
 		}
 	}
 
-	public function send_published( $post_ID, $post, $update ) {
+	public function send_published( $post_ID = null, $post = null, $update = null ) {
```

Comment:
As per my previous comment, I don't think you need to check all these values, just:

```php
public function send_published( $post_ID, $post = null, $update = null ) {
  if ( is_null( $post ) ) {
    return;
  }
}
```